Formula/ssh-agent-filter.rb

```diff
@@ -0,0 +1,62 @@
+class SshAgentFilter < Formula
+  desc "filtering proxy for ssh-agent"
+  homepage "https://github.com/tiwe-de/ssh-agent-filter"
+  url "https://github.com/tiwe-de/ssh-agent-filter/archive/0.4.2.tar.gz"
+  sha256 "5586e40da34f4afd873d7ff30f115c4a1b9455c1b81f3c4679c995db7615b080"
+
+  depends_on "help2man" => :build
+  depends_on "boost"
+  depends_on "nettle"
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project as a pull request and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!